modules/navigation/godot_navigation_server_2d.cpp

```diff
@@ -141,6 +141,13 @@ static Transform3D trf2_to_trf3(const Transform2D &d) {
 	return Transform3D(b, o);
 }
 
+static Transform2D trf3_to_trf2(const Transform3D &d) {
```

Comment:
When tested locally with the following GDScript, I get output that seems to be within epsilon.
```
extends Node2D


var region: RID = NavigationServer2D.region_create()
var t = Transform2D()
var rot = -0.5 # The rotation to apply.

# Called when the node enters the scene tree for the first time.
func _ready():
	t.origin = Vector2(350,150)
	t.x.x = cos(rot)
	t.y.y = cos(rot)
	t.x.y = sin(rot)
	t.y.x = -sin(rot)
	# Scale
	t.x *= 3
	t.y *= 3
	print(t)
	NavigationServer2D.region_set_transform(region, t)
	pass # Replace with function body.


# Called every frame. 'delta' is the elapsed time since the previous frame.
func _process(delta):
	print(t, NavigationServer2D.region_get_transform(region))
	pass

OUTPUT: 
[X: (2.632748, -1.438277), Y: (1.438277, 2.632748), O: (350, 150)]
[X: (2.632751, -1.438278), Y: (1.438263, 2.632751), O: (350, 150)]
```

Reply:
That is fine, a small difference is to be expected from converting back and forth between 2D and 3D.